tensorflow/tools/api/golden/v1/tensorflow.pbtxt

```diff
@@ -1836,6 +1836,10 @@ tf_module {
     name: "quantized_concat"
     argspec: "args=[\'concat_dim\', \'values\', \'input_mins\', \'input_maxes\', \'name\'], varargs=None, keywords=None, defaults=[\'None\'], "
   }
+  member_method {
+    name: "quantized_mat_mul_with_bias_and_requantize"
```

Comment:
The new op needs to have visibility:HIDDEN in its api_def file since we don't want tf.quantized_mat_mul_with_bias_and_requantize to be a public TF symbol

Reply:
@alextp Agree and added! api_def test passes on internal testing!

Reply:
Can you rerun the api_compatibility_test to regenerate the goldens then?

Reply:
@alextp Yes done!